### PR TITLE
Use /sytsem/v1/metrics base route for metrics serivce

### PR DESCRIPTION
**Pull request by malnick**

### CODE REVIEW

**Comment by alberts** [comment]
please squash

File: nginx.master.conf [hunk 1]
```diff
@@ -297,11 +297,13 @@ http {
 
             proxy_pass http://$agentaddr:61001/system/v1/$url$is_args$query_string;
         }
-
-        location /system/metrics {
+        
+        location /system/v1/metrics/ {
             access_by_lua 'auth.validate_jwt_or_exit()';
-            include common/proxy-headers.conf;
```
Comment (alberts) [inline nginx.master.conf]: why did we stop using this include? is anything else still using this file?

**Comment by alberts** [comment]
also, please fix typo in commit message

File: nginx.agent.conf [hunk 2]
```diff
@@ -42,10 +42,12 @@ http {
 
             proxy_pass http://log/;
         }
-
-        location /system/metrics {
-            include common/proxy-headers.conf;
-            proxy_pass http://metrics;
+        
+        location /system/v1/metrics/ {
```
Comment (alberts) [inline nginx.agent.conf]: we're renaming endpoints... are we sure no customer might be using there?

Comment (malnick) [inline nginx.agent.conf]: These are new services, so no customers are changing them. We want to version the /system/ URI to make it feel more like a single API.

**Comment by malnick** [comment]
@alberts Squashed, spelling corrected.

File: nginx.agent.conf (hunk 2 above)
Comment (spahl) [inline nginx.agent.conf]: Correct

**Pull request merged**